guides/vpn/README.md

```diff
@@ -37,39 +37,60 @@ That is it! You should now be connected to the `Made Tech - VPN`. If you are exp
 
 ### Ubuntu/Debian
 
-https://docs.foxpass.com/docs/foxpass-linux-l2tpipsec
+1. Install the L2TP module for network-manager & gnome extension if necessary
+    ```bash
+    sudo apt update
+    sudo apt install network-manager-l2tp
+
+    # If in a gnome environment
+    sudo apt install network-manager-l2tp-gnome
+    ```
+2. Follow the steps to add the VPN connection below.
+3. You must set the Phase1 and Phase2 algorithms in IPSec Settings:
+   1. Open the Made Tech VPN connection settings
+   2. Navigate to Identity tab
+   3. Open IPsec Settings
+   4. Under Advanced:
+      - Phase1 Algorithms `3des-sha1-modp2048`
+      - Phase2 Algorithms `3des-sha1`
+   5. Save, close
+4. Connect & enter password
```

Comment:
Is there a reason for adding all of this which mirrors the content in the link? Foxpass is what we are using, so to me, it makes more sense to link to their docs directly.

The only difference I can see here are the phase algorithms being switched around.